Packages/com.unity.inputsystem/InputSystem/Plugins/PS4/DualShockGamepadPS4.cs

```diff
@@ -315,11 +344,22 @@ public int ps4UserId
         {
             get
             {
-                UpdatePadSettingsIfNeeded();
+#if !UNITY_2019_1_OR_NEWER
+                UpdatePadSettingsIfNeeded(); // 2018.2 uses IOCTL to read this. not required in later versions
+#endif
                 return m_PS4UserId;
             }
         }
 
+        public bool IsAimController
```

Comment:
Properties need to follow the UnityEngine convention, i.e. `lowerCaseStart`. We'll clean it up in the consistency pass.